CSS/CSS.sublime-settings

```diff
@@ -8,4 +8,6 @@
 
     // Default separators except `-`
     "word_separators": "./\\()\"':,.;<>~!@#$%^&*|+=[]{}`~?",
+
+    "sub_word_separators": "_-",
```

Comment:
A comment denoting the difference to the default would be appreciated, similar to the `word_separators` setting, so that it can be synchronized more easily in the future when the default changes. (And gives some context to the change.)